db/mysql_structure_init.sql

```diff
@@ -4,6 +4,7 @@
 -- mysql -u sdrAdmin -p --default-character-set=utf8 archive_catalog_development < db/mysql_structure_init.sql
 
 SET FOREIGN_KEY_CHECKS=0;
+SET SQL_MODE='ALLOW_INVALID_DATES';
```

Comment:
That sounds fine, lets just add a comment so people understand why this is here.